app/models/name.rb

```diff
@@ -1934,7 +1935,7 @@ def self.format_sort_name(name, author)
                     gsub(/[Đđ]/, "d"). # mysql isn't collating these right
                     gsub(/[Øø]/, "O").
                     strip.
-                    sub(/^group$/, " group")
+                    sub(/^(group|clade)$/, "\1")
```

Comment:
Oops!  The space was deliberate.  Space comes before all other nonblank characters, so adding the space -- that is, e.g., "Foo<space><space>group" -- would ensure that "Foo group" sorts before "Foo Author".  If this passed tests, it means that this case (e.g., sort the names "Foo group", "Foo clade" and "Foo Author") is not tested.  Please add a space before the slash-one: "<space>\1".